app/views/doctors/search_root.html.haml

```diff
@@ -0,0 +1,4 @@
+%header.site-header
+  %h1.site-title= t(:app_purpose)
+  = render "doctors/doctor_search_form"
+  = button_tag t("get_location.submit_text"), type: "submit", name: nil, id:'get_location'
```

Comment:
I've tried this now, and I don't think it will work, unless we can get JS to handle the language.
Since Rails is handling the language stuff, I think the best way to do this is to have it in rails and activate+reveal the button in JavaScript. Doing that with JS requires us to keep knowledge in two places potentially.